core/src/services/oss/core.rs

```diff
@@ -186,6 +187,53 @@ impl OssCore {
         Ok(req)
     }
 
+    /// Oss append object request
+    ///
+    /// # Note
+    ///
+    /// This request is used to append data to an existing object or create an appendable object.
+    /// So you must set the `append` and `position` header.
+    ///
+    /// https://www.alibabacloud.com/help/object-storage-service/latest/appendobject
+    pub fn oss_append_object_request(
+        &self,
+        path: &str,
+        position: u64,
+        size: usize,
+        args: &OpAppend,
+        body: AsyncBody,
+    ) -> Result<Request<AsyncBody>> {
+        let p = build_abs_path(&self.root, path);
+        let endpoint = self.get_endpoint(false);
+        let url = format!("{}/{}", endpoint, percent_encode_path(&p));
+
+        let mut req = Request::post(&url);
+
+        // The header `append` does not need a value.
+        req = req.header(HeaderName::from_static("append"), "");
+        req = req.header(HeaderName::from_static("position"), position);
```

Comment:
I believe they are not in headers.

Reply:
No, the documentation must be incorrect. The terms `append` and `position` are query-related and the documentation does not accurately reflect this information.

Reply:
My bad. This is part of url. 🥲